backend/src/main/java/gov/cdc/usds/simplereport/db/model/TestOrder.java

```diff
@@ -23,19 +26,27 @@
 public class TestOrder extends BaseTestInfo {
 
 	@ManyToOne(fetch = FetchType.LAZY)
-	@JoinColumn(name = "patient_answers_id" )
+	@JoinColumn(name = "patient_answers_id")
 	private PatientAnswers askOnEntrySurvey;
 	@Column
 	private LocalDate dateTested; // REMOVE THIS COLUMN
-	@Column
-	private Date dateTestedBackdate;
 	@Column(nullable = false)
 	@Type(type = "pg_enum")
 	@Enumerated(EnumType.STRING)
 	private OrderStatus orderStatus;
-	@OneToOne(optional = true)
-	@JoinColumn(name="test_event_id")
-	private TestEvent testEvent;
+
+    // strictly speaking, this is no longer OneToOne since corrections could have more than one,
+    // but this is kept up-to-date with the latest one.
+    @Column(columnDefinition = "uuid")
+    private UUID testEventId;    // id used directly without needing to load
+
+    // join with test_event just to access created_at. Cannot figure out how just get one column
+	// It's separate from testEventId because we eventually want to get rid of it.
+	@Transient
```

Comment:
This seems really wonky and unhappy to me, let's talk about how to make it not that.

Reply:
The goal was to intentionally break getting to a TestEvent by jumping through TestOrder. 

But now that TestEvent contains a copy of the `AskOnEntrySurvey` it's no longer required and it's been removed.